pkg/p2p/libp2p/internal/handshake/pb/handshake.proto

```diff
@@ -16,6 +16,7 @@ message Ack {
     BzzAddress Address = 1;
     uint64 NetworkID = 2;
     bool Light = 3;
+    string WelcomeMessage  = 4;
```

Comment:
🌷  Just one suggestion guys. I would put this on some high random index, not to intervene with the protocol, as we might change versions of the protocols and update the message, so we don't need to have a reserved field in the lower register.  So maybe something like: 
```suggestion
    string WelcomeMessage  = 99;
```

Reply:
@pradovic Should we consider reserving some of the low field numbers?

Reply:
hmm, we can, but I don't see the need yet tbh. Reserving is useful when deleting and obsoleting fields, so they are not reused for some other fields. In this case I don't think we need to do it.

Reply:
So for example, if we end up removing some field, or `WelcomeMessage` field, then it would be good to reserve this one so it is not reused for something else

Reply:
Thanks @pradovic, change commited!